### PR TITLE
Add jug recipe

### PR DESCRIPTION
**Pull request by luispedro**
Python only package:

https://jug.readthedocs.io

### CODE REVIEW

File: recipes/jug/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/jug/meta.yaml]: It looks like this package has a python entry point:

https://github.com/luispedro/jug/blob/master/setup.py#L69-L72

please add it as in:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/jug/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - python
+    - six
+    - setuptools
+  run:
+    - python
+    - six
+    - bottle
+    - yaml
+
+test:
```
Comment (synapticarbors) [inline recipes/jug/meta.yaml]: Please test the command line script:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/jug/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - six
+    - setuptools
+  run:
```
Comment (synapticarbors) [inline recipes/jug/meta.yaml]: Is `redis` a runtime requirement:

https://github.com/luispedro/jug/blob/master/setup.py#L76

File: recipes/jug/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    - python
+    - six
+    - bottle
+    - yaml
```
Comment (synapticarbors) [inline recipes/jug/meta.yaml]: Should this be `pyyaml` instead of `yaml`. The latter is the C library.

File: recipes/jug/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - six
```
Comment (synapticarbors) [inline recipes/jug/meta.yaml]: is `six` required to build the package?

Comment (luispedro) [reply recipes/jug/meta.yaml]: Good catch. removed.

File: recipes/jug/meta.yaml (hunk 3 above)
Comment (luispedro) [reply recipes/jug/meta.yaml]: It's an optional requirement.

If you use the redis integration, then yes. But I actually rarely use it myself (although I know others who do).

Comment (synapticarbors) [reply recipes/jug/meta.yaml]: It seems fine to just leave it out then.

File: recipes/jug/meta.yaml (hunk 2 above)
Comment (luispedro) [reply recipes/jug/meta.yaml]: Done in update.

File: recipes/jug/meta.yaml (hunk 1 above)
Comment (luispedro) [reply recipes/jug/meta.yaml]: Done in update.

**Comment by mwcraig** [comment]
Thanks, @luispedro for the contribution. Merging, and shortly (depending on travis backlog) a feedstock should be created shortyl.

**Pull request merged**